src/Relation/Binary/Construct/Closure/Reflexive/Properties.agda

```diff
@@ -0,0 +1,59 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties of reflexive closures
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K #-}
+
+module Relation.Binary.Construct.Closure.Reflexive.Properties where
+
+open import Data.Product as Prod
+open import Function
+open import Relation.Binary
+open import Relation.Binary.Construct.Closure.Reflexive
+import Relation.Binary.PropositionalEquality as PropEq
+open PropEq using (_≡_; refl)
+open import Relation.Nullary
+
+module _ {a b ℓ} {A : Set a} {B : Set b} {_~_ : Rel A ℓ} where
+    =[]⇒ : ∀ {p q} {P : Rel A p} {Q : Rel B q} {f} → P =[ f ]⇒ Q → Refl P =[ f ]⇒ Refl Q
+    =[]⇒ x [ x∼y ] = [ x x∼y ]
+    =[]⇒ x refl = refl
```

Comment:
Please could you try and align the `=` sign where possible, makes it much easier to read.

Reply:
Done